packages/mdc-toolbar/README.md

```diff
@@ -2,8 +2,21 @@
 
 MDC Toolbar acts as a container for multiple rows containing items such as
 application title, navigation menu, and tabs, among other things. Toolbars
-scroll with content by default, but supports fixed on top as well. Currently,
-this component does not yet support the "Waterfall" or "Flexible Header" patterns.
+scroll with content by default, but supports fixed and waterfall behavior as well.
+
+When using the **waterfall** pattern, a toolbar will have no elevation when the
+page is scrolled to the top, and gain elevation when a user begins to scroll
+down the page. When using the **fixed** pattern, a persistent elevation is added
```

Comment:
I don't think so. The second row in [usage](https://material.io/guidelines/components/toolbars.html#toolbars-usage) shows `position:relative` don't have elevation as they scroll with the page.